frontend/src/components/ToDoLists.js

```diff
@@ -0,0 +1,24 @@
+import { useState } from 'react';
+import { getList } from '../http/lists'
+
+export const ToDoLists = ({lists, listState, setListState}) => {
+// when the list button is clicked the API will return a list item and
+// we want to set that list item to a state which will then be passed up
+// this will allow us to render the current list item onto the main page
+    const handleListClick = async(id) => {
+        const list = await getList(id);
+        if (list.ok) {
+            setListState(list)
+        } else {
+            return list.error
```

Comment:
Why are we returning list.error? The result of this function shouldn't be a return should it? If we fail to get a list we should set some sort of state, no?

Also, reading that we are passing the state back up makes me think maybe we should have a component that wraps both the todo lists and the todo detail component. Just food for though, no need to do it in this PR.